website/docs/providers/ldap/index.md

```diff
@@ -102,7 +102,15 @@ In this mode, the outpost will always execute the configured flow when a new bin
 
 This mode uses the same logic as direct bind, however the result is cached for the entered credentials, and saved in memory for the standard session duration. Sessions are saved independently, meaning that revoking sessions does _not_ remove them from the outpost, and neither will changing a users credentials.
 
-## Search Modes
+## Searching & Search Modes
+
+Any user that is authorized to access the LDAP Provider's application can execute search the LDAP directory. Without explicit permissions, users search request will return info about themselves, including user info, group info and group membership.
+
+[Users](../../user-group-role/user/index.mdx) and [Roles](../../user-group-role/roles/index.mdx) can be assigned the permission "Search full LDAP directory" to allow them to search the full LDAP directory and retrieve information about all users in the authentik instance.
+
+:::info
+Up to authentik version 2024.8 this was managed using the "Search group" attribute in the LDAP Provider, where users could be added to a group to grant them this permission. With authentik 2024.8 this is automatically migrated to the "Search full LDAP directory" permission, which can be assigned more flexibly.
```

Comment:
So there is no action that users who upgrade to 2024.8 and later need to take, right?

Reply:
Correct

Reply:
Please note many of us lost a lot of time on this https://github.com/goauthentik/authentik/issues/11548#issuecomment-2429442922 since we did not get we had to work on `object-level permission`